astroquery/casda/core.py

```diff
@@ -222,6 +237,87 @@ def stage_data(self, table, verbose=False):
 
         return fileurls
 
+    def stage_data(self, table, verbose=False):
+        """
+        Request access to a set of data files. All requests for data must use authentication. If you have access to the
+        data, the requested files will be brought online and a set of URLs to download the files will be returned.
+
+        Parameters
+        ----------
+        table: `astropy.table.Table`
+            A table describing the files to be staged, such as produced by query_region. It must include an
+            access_url column.
+        verbose: bool, optional
+            Should status message be logged periodically, defaults to False
+
+        Returns
+        -------
+        A list of urls of both the requested files/cutouts and the checksums for the files/cutouts
+        """
+        if not self._authenticated:
+            raise ValueError("Credentials must be supplied to download CASDA image data")
+
+        if table is None or len(table) == 0:
+            return []
+
+        job_url = self._create_job(table, 'async_service', verbose)
+
+        return self._complete_job(job_url, verbose)
+
+    def cutout(self, table, radius=1*u.arcmin, verbose=False, **kwargs):
```

Comment:
I'm sorry, I think my previous comment was confusing. I meant that kwargs to be kwarg only (enforced by the `*` in the signature, not that they should be hidden in a `**kwargs`. 
```suggestion
    (self, table, *, coordinates=None, radius=None, height=None, width=None, band=None, channel=None, verbose=False)
```